index.html

```diff
@@ -129,14 +183,19 @@ <h1 class="h-text"> JavaScript Event Loop</h1>
 
         <div class="body">
             <section class="right-body">
-                <div class="button">
+                <!--    <div class="button">
                     <button>Edit</button>
                     <button>Return</button>
                     <button>Pause</button>
                     <button>Resume</button>
-                </div>
+                </div> -->

```

Comment:
Are we removing these buttons? I think, we should keep these buttons.

Reply:
no we are not going to remove it instead we have to add a new button and on click these buttons will appear.